### DISCUSSION TITLE
Backup not working / rAudio  20220904

### DISCUSSION THREAD

**Discussion by sp300b** [rern/rAudio#1007] [Fixed Issues]
Hi, 
I have noticed that somehow Backup is not working (neither in 20220902 nor 20220904 versions). 
Please check, attached backup file.

Thanks a lot in advance.
[backup.gz](https://github.com/rern/rAudio-1/files/9484829/backup.gz)

**Comment by rern** [comment]
It's a new bug. To be fixed with next update.
In the meantime:
```sh
sed -i 's/webradio/tmp &/' /srv/http/bash/common.sh
```

**Reply by sp300b** [reply]
Working! Thanks a lot, Rern! Keep up the great work.
